ethpm_types/contract_type.py

```diff
@@ -241,11 +241,13 @@ def validate_uri(cls, uri):
     @classmethod
     def validate_genesis_hash(cls, uri):
         genesis_hash, _, _ = uri.replace("blockchain://", "").split("/")
-        assert is_valid_hash(genesis_hash), f"hash is not valid: {genesis_hash}"
+        assert is_valid_hex("0x" + genesis_hash), f"hash is not valid: {genesis_hash}"
```

Comment:
https://bandit.readthedocs.io/en/latest/plugins/b101_assert_used.html

Reply:
Yeah should flip these to `if ...: raise` statments

Reply:
30350a9